config/locales/fr.yml

```diff
@@ -11,7 +11,14 @@ fr:
     formats:
       default: "%b %d, %Y"
 
+   helpers:
```

Comment:
The problem was the indentation right here, there was one space missing. The translation file uses indentation to create a hierarchy of keys. But yeah, that error message was not helpful.